docs/SUPPORT.md

```diff
@@ -1,9 +1,11 @@
 # Getting Support
 
-Laminas offers the following support channels:
+Laminas offers three support channels:
```

Comment:
This is how we have now in migration template, but I think I prefer the previous version. 
What do you think, @weierophinney?

Reply:
I like "the following" versus "three", because if the quantity of support channels changes, we don't have to change both the list _and_ this preamble.